charts/rancher-eks-operator/1.0.801/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.6-alpha1
```

Comment:
The chart will work on 2.5.6+ because `additionalTrustedCAs` is false by default. Even if a user sets it to `true` when deploying the helm chart, the chart will deploy successfully as long as the secret exists as in the Rancher docs.

Reply:
Okay, should be fine. I'm not sure if we have stances on when we should let older versions of rancher use newer charts. @cbron do you have any insights on that?

Reply:
It depends on the chart. If there are no UI/API changes then it can go into previous rancher versions, but to be safe we can always min the new one.

Reply:
Done.